packages/atomic/src/components/atomic-result/atomic-result.pcss

```diff
@@ -1,5 +1,236 @@
-@import "tailwindcss/base";
+@import 'tailwindcss/base';
```

Comment:
This CSS file is a bit ugly. Tailwind includes some doc on adding a [nesting library](https://tailwindcss.com/docs/using-with-preprocessors#nesting), if you prefer I can add that.

Reply:
Yes :)